src/ua/griddynamics/Cell.java

```diff
@@ -0,0 +1,37 @@
+package ua.griddynamics;
+
+public class Cell {
+    boolean mine;
+    boolean checked;
+    boolean markedAsMine;
+    boolean markAsFree;
```

Comment:
Why is `markAsFree` needed as we have` isEmpty()` function

Reply:
Done